generators/app/index.js

```diff
@@ -3,20 +3,24 @@
  *--------------------------------------------------------*/
 'use strict';
 
+//TODO:
+// - copy over Jupyter Book files
```

Comment:
remove if you're done with change. if it's for future reference then please add task